examples/medplum-provider/src/components/encountertasks/TaskQuestionnaireForm.tsx

```diff
@@ -68,6 +68,103 @@ export const TaskQuestionnaireForm = ({ task, onChangeResponse }: TaskQuestionna
           onChange={onChange}
         />
       )}
+
+      {notFound && <NotFoundPanel />}
```

Comment:
You could then consider using the `<OperationOutcomeAlert>` component

```suggestion
      {outcome && <OperationOutcomeAlert outcome={outcome} />}
```

That would then handle other error codes.

Not a big deal, for consideration.  It's nice to have common utilities rather than having to rebuild one-off components for this.